doc/cephadm/services/snmp-gateway.rst

```diff
@@ -2,10 +2,10 @@
 SNMP Gateway Service
 ====================
 
-SNMP_ is still a widely used protocol, to monitor distributed systems and devices across a variety of hardware
+`SNMP`_ is still a widely used protocol, to monitor distributed systems and devices across a variety of hardware
```

Comment:
Why the underscores here vs :ref: ?

Also

>is a widely used protocol for monitoring distributed systems and devices.

Reply:
A `:ref:` would need a label, it is for internal links (other RST docs).  
This is using the "link definitions" feature to an "external link".

Will fix the language.